server/lib/routes/get-index.js

```diff
@@ -3,13 +3,18 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var flowMetrics = require('../flow-metrics');
+var uaParser = require('node-uap');
 
 module.exports = function (config) {
   var ALLOWED_PARENT_ORIGINS = config.get('allowed_parent_origins');
   var AUTH_SERVER_URL = config.get('fxaccount_url');
   var CLIENT_ID = config.get('oauth_client_id');
   var ENV = config.get('env');
   var FLOW_ID_KEY = config.get('flow_id_key');
+  var FIREFOX_IOS_DOWNLOAD_LINK = 'https://app.adjust.com/2uo1qc?' +
```

Comment:
Rereading the Apple docs, I don't think we need to include this for now.

https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html

> app-argument: (Optional.) A URL that provides context to your native app. If you include this, and the user has your app installed, she can jump from your website to the corresponding position in your iOS app.

Possibly after Universal Links and Deep Links support have been push to production mobile apps?